### PR TITLE
Fix expiresIn (convert String to Int)

### PR DESCRIPTION
**Pull request by RaphaelStAlme**
I detected an issue when i try to get acess token from adb2c. The reason is the "expires_token" variable is didn't converted to int, he stay in String type. That why i fixed this problem in integrate this code.

### CODE REVIEW

**Review by RaphaelStAlme — COMMENTED** [review]
fix: add double.tryParse(expiresIn)!.toInt(); instead of add double.tryParse(expiresIn).toInt();

File: lib/src/handle_access_token_response.dart [hunk 1]
```diff
@@ -68,9 +68,18 @@ Credentials handleAccessTokenResponse(http.Response response, Uri tokenEndpoint,
...
-      throw FormatException(
-          'parameter "expires_in" was not an int, was "$expiresIn"');
+    if (expiresIn != null) {
+      if (expiresIn is String) {
+        try {
+          expiresIn = double.tryParse(expiresIn)!.toInt();
+        } catch (e) {
+          throw FormatException(
+              'parameter "expires_in" was not an int, and given string cant not be parsed to int : "$expiresIn"');
+        }
```
Comment (jonasfj) [inline lib/src/handle_access_token_response.dart:78]:
```suggestion
        try {
          expiresIn = double.parse(expiresIn).toInt();
        } on FormatException {
          throw FormatException(
              'parameter "expires_in" was not an int, and given string cannot be parsed to int : "$expiresIn"');
        }
```

**Review by jonasfj — APPROVED** [review]
LGTM, can we fix the typo in the exception message?

File: lib/src/handle_access_token_response.dart (hunk 1 above)
Comment (jonasfj) [reply lib/src/handle_access_token_response.dart:78]: * [ ] Typo `cant not`
 * [ ] Why not just do `on FormatException`, then we don't need the null-check operator...

Comment (jonasfj) [reply lib/src/handle_access_token_response.dart:78]: I fixed this and merged :D

**Pull request merged**